tests/acceptance/request/config-test.js

```diff
@@ -183,11 +183,15 @@ module('Acceptance | request/config', function (hooks) {
       assert.equal(page.config[0].source, '.travis.yml');
     });
 
-    test('hides the tab when no config is found', async function (assert) {
-      this.request.raw_configs = [];
-
-      await visit(`/travis-ci/travis-web/jobs/${this.job.id}`);
-      assert.ok(page.configTab.isDisabled, 'expected the config tab to be disabled when there’s no .travis.yml');
-    });
+    // TODO decide how to handle this case going forward. we want to display config validation
+    // results publicly, but old builds won't have any messages. maybe it's alright to mark a
+    // build config as valid if validation has never run?
+    //
+    // test('hides the tab when no config is found', async function (assert) {
+    //   this.request.raw_configs = [];
+    //
+    //   await visit(`/travis-ci/travis-web/jobs/${this.job.id}`);
+    //   assert.ok(page.configTab.isDisabled, 'expected the config tab to be disabled when there’s no .travis.yml');
+    // });
```

Comment:
this status still is "undecided" :) yes, i think having a discussion about this would be a good thing. how would we best facilitate that? just pinging the web team on slack?

Reply:
That's a good question. Yeah I would say either ping the web team on slack or ping individuals here. Since there's multiple TODOs on this PR, maybe a ping in slack is the way to go. Although, with a couple of PRs after this one yet to be reviewed, I also wonder what is the best timing for such a discussion. WDYT @artursmirnov @piotr-travisci ?

Reply:
i have added that question to a new section "to be discussed/resolved" on https://github.com/travis-ci/travis-web/pull/2399 in order to keep track of this

Reply:
Sounds good, thanks!